components/infoboxes/commons/infobox_team.lua

```diff
@@ -63,7 +64,7 @@ function Team:createInfobox(frame)
 	local history = Team:getHistory(infobox, args)
 
     infobox :header('Links', not Table.isEmpty(links))
-            :links(links)
+            :links(links, 'team')
```

Comment:
_LINK_VARIANT_TEAM

Reply:
how about `_LINK_VARIANT = 'team'` at the top and then
`:links(links, _LINK_VARIANT)` in that line?

Reply:
Yes, sure